tests/integration/telemetry/stackdriver/stackdriver_filter_test.go

```diff
@@ -136,6 +136,7 @@ func TestStackdriverMonitoring(t *testing.T) {
 
 func TestMain(m *testing.M) {
 	framework.NewSuite(m).
+		Skip("https://github.com/istio/istio/issues/29268").
```

Comment:
Given this can be forgotten by us, we could create a Maistra Jira issue and link it with the upstream issue, and reference the maistra issue here.

Reply:
the linter verifies that this specifies an istio/istio issue, that's why I put that there

Reply:
verifies in what sense? My concern is that this issue is fixed upstream and we never re-enable this test because it went unnoticed. We could at least create a Jira issue so that we can revisit it in the future.

Reply:
it just checks that the skipReason contains an istio/istio issue. sure we can track it in a MAISTRA ticket